### PR TITLE
implemented Plain-text input element

### PR DESCRIPTION
**Pull request by nurul-umbhiya**
Reference: https://api.slack.com/reference/block-kit/block-elements#input

### CODE REVIEW

File: src/Inputs/TextInput.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+use Jeremeamia\Slack\BlockKit\Exception;
+use Jeremeamia\Slack\BlockKit\Partials\PlainText;
+
+class TextInput extends InputElement
+{
+
+    /** @var PlainText */
+    private $placeholder = '';
+
+    /** @var PlainText */
```
Comment (jeremeamia) [inline src/Inputs/TextInput.php]: Type should be `string`

File: src/Inputs/TextInput.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    private $initialValue = '';
+
+    /** @var Boolean */
```
Comment (jeremeamia) [inline src/Inputs/TextInput.php]: Type should be `bool`

File: src/Inputs/TextInput.php [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    private $multiline = false;
+
+    /** @var unsigned */
```
Comment (jeremeamia) [inline src/Inputs/TextInput.php]: Type should be `int`

File: src/Inputs/TextInput.php [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    private $minLength;
+
+    /** @var unsigned */
```
Comment (jeremeamia) [inline src/Inputs/TextInput.php]: Type should be `int`

File: src/Inputs/TextInput.php [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    private $maxLength = 3000;
```
Comment (jeremeamia) [inline src/Inputs/TextInput.php]: I don't want to set this, since it is an optional value.

**Review by jeremeamia — COMMENTED** [review]
Hey, thanks for contributing this. It looks good in general, but we should clean up a few small things.

File: src/Inputs/TextInput.php (hunk 1 above)
Comment (nurul-umbhiya) [reply src/Inputs/TextInput.php]: Fixed.

File: src/Inputs/TextInput.php (hunk 3 above)
Comment (nurul-umbhiya) [reply src/Inputs/TextInput.php]: fixed

File: src/Inputs/TextInput.php (hunk 5 above)
Comment (nurul-umbhiya) [reply src/Inputs/TextInput.php]: fixed

**Review by jeremeamia — COMMENTED** [review]
Thanks!

**Review by jeremeamia — APPROVED** [review]

**Comment by jeremeamia** [comment]
@nurul-umbhiya I did some follow-up changes after merging your PR to add tests: https://github.com/jeremeamia/slack-block-kit/commit/ee4861ef6f2228a89d3fdadd208419e96312f3d3

**Comment by nurul-umbhiya** [comment]
Thank you very much. I urgently needed this feature for a project so I stepped in to implement this by myself. Honestly, I spent like an hour or so, after I initially succeeded I moved backed to my current task. So I couldn't test the whole integration. Thanks again for this awesome project. It really saved a lot of time.

**Pull request merged**